.github/workflows/examples.yml

```diff
@@ -41,28 +41,28 @@ jobs:
       - run: npm install wasm-opt -g
 
       - name: Checkout sources & submodules
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
         with:
           fetch-depth: 1
           submodules: recursive
 
       - name: Install toolchain
         id: toolchain
-        uses: actions-rs/toolchain@master
+        uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: ${{ matrix.toolchain }}
           components: rust-src
           override: true
 
       - name: Install cargo-dylint
-        uses: baptiste0928/cargo-install@v1
+        uses: baptiste0928/cargo-install@bf6758885262d0e6f61089a9d8c8790d3ac3368f # v1.3.0
```

Comment:
Can we do the following instead?

```suggestion
        uses: baptiste0928/cargo-install@1.3.0
```

Reply:
Highly undesirable. Preferable way to pin GitHub actions to a hash. At least we are striving to do that for vendors without strong background to avoid situations like this https://gitlab.parity.io/parity/vulnerability-response/-/issues/50

Reply:
It's almost the same thing though, isn't it? The `v1.3.0` refers to a specific tag, which is like pointing to a specific commit. Either way, I'm not too fussy about this, so if you think the commit hash is better let's roll with that

Reply:
Almost, but tag can be easily moved to another commit. With hash such things a bit harder :)